### ISSUE TITLE
Duplicate instance Profunctor FUN with linear-base 0.2.0

### ISSUE THREAD

**Issue by Lysxia** [sjoerdvisscher/one-liner#18]
linear-base 0.2.0 adds a `Profunctor FUN` instance, which now conflicts with one-liner:

```
src/Generics/OneLiner/Classes.hs:90:10: error:
    Duplicate instance declarations:
      instance Profunctor (FUN 'One)
        -- Defined at src/Generics/OneLiner/Classes.hs:90:10
      instance Profunctor (FUN 'One)
        -- Defined in ‘Data.Profunctor.Linear’
   |
90 | instance Profunctor (FUN 'One) where
   |          ^^^^^^^^^^^^^^^^^^^^^
```

**Comment by sjakobi** [comment]
As a Hackage trustee I have created a revision for `one-liner-2.0`: https://hackage.haskell.org/package/one-liner-2.0/revisions/

**Comment by sjoerdvisscher** [comment]
Fixed with v2.1. Thanks for reporting, and sorry for the late response!
